src/cfg/wto.hpp

```
@@ -105,10 +105,22 @@ class WtoCycle final {
     }
 
     void for_each_loop_head(auto&& f) const {
```

Clang (14.0.6)

[warning] method 'for_each_loop_head' can be made static

(readability-convert-member-functions-to-static)